bin/generateEasyConfig.R

```diff
@@ -59,35 +59,47 @@ repos$bioconductor$active = c('http://www.bioconductor.org/packages/release/bioc
 #    'NLopt'='2.6.1'
 #)
 #
-# This list is for R 4.0.x with foss 2018b:
+# This list is for R 4.2.x with foss 2022a:
 #
 dependency_defaults <-list(
-    'pkg-config'='0.29.2',
-    'libreadline'='8.0',
-    'ncurses'='6.2',
+    'pkgconf'='1.8.0',
+    'Xvfb'='21.1.3',
+    'Autotools'='20220317',
+    'R'='4.2.1-bare',
```

Comment:
If you don't put R in the dependency list here it will not show up in final eb file

Reply:
The R version is always added to the list op dependencies. See lines:
```
    writeLines("dependencies = [", fh)
    writeLines("    ('R', '%(version)s', '-bare'),", fh)
```
It does not make sense to add it here as you must always have an R version loaded in your environment and for which you want to make the EasyConfigs: there is no default and it cannot be empty.